logger.ts

```diff
@@ -1,30 +1,46 @@
 import * as winston from 'winston';
 import {environment} from './config/globals';
 
+
+
+const errorFormat = winston.format.combine(
+    winston.format.errors({ stack: true }),
+    winston.format.timestamp({
+        format: 'YYYY-MM-DD HH:mm:ss'
+    }),
+    winston.format.json()
+)
+
+const consoleFormat = winston.format.combine(
+    winston.format.errors({ stack: true }),
+    winston.format.colorize({ message: true }),
+    winston.format.splat(),
+    winston.format.printf(({message, stack}) => {
+        const timeStamp = new Date().toTimeString().split(' ')[0];
+        if (stack)
+            return `[${timeStamp}] ${message} \n${stack}`;
+        return `[${timeStamp}] ${message}`;}),
+    
+) 
+
 const logger = winston.createLogger({
-    format: winston.format.json(),
+    format: errorFormat,
     transports:[
-        // writes all logs with level 'error' to the error.log
+        // writes all logs with level 'warn' or higher ('error' too) to the error.log
         new winston.transports.File({
             filename: 'logs/error.log',
-            level: 'error',
-            maxsize: 5242880, // 5MB
+            level: 'warn',
```

Comment:
Maybe there should be a possibility to set this dynamically with a npm script setting an ENV variable